plugins/hydra_joblib_launcher/hydra_plugins/hydra_joblib_launcher/_core.py

```diff
@@ -50,6 +50,18 @@ def execute_job(
     return ret
 
 
+def process_joblib_cfg(joblib_cfg: Dict[str, Any]) -> Dict[str, Any]:
+    for k, v in joblib_cfg.items():
+        if k in ["pre_dispatch", "batch_size", "max_nbytes"] and v:
+            try:
+                joblib_cfg[k] = int(v)
+            except ValueError:
+                joblib_cfg[k] = v
+        else:
```

Comment:
This `else` is not required

Reply:
thx, yea initially I wanted to create a new dict here which is why I looped all the items.

Took your suggestions and simplied the method now.